backend/app.py

```diff
@@ -0,0 +1,22 @@
+from flask_restful import Api
+from flask_jwt_extended import JWTManager
+import secrets
+import backend
+
+# API description in swagger - https://app.swaggerhub.com/apis/DoRTaL94/UserManagment/1.0.0
+
+app = backend.app
+# Creates all tables in app. Table's name defined with __tablename__ variable.
+backend.db.create_all()
```

Comment:
It may not be a good idea to do this every time you start your app. Consider what would happen in production when your database contains production information and your app restarts from some reason...

Reply:
db.create_all() creates only tables that not exists in the db. If a table exists in the db it won't be changed.

Reply:
What would be the process for updating the structure of an existing table, when you decide you need to add a field to a model class for example?

Reply:
Well, currently we have two choices that I can think of:
1. Delete the relevant table, and when db.create_all being called again it will create the updated table with the new column. **(Good only for development)**
2. New field = new column in the table, so we could add the new column manually by connecting to PostgreSQL from the command line.

For more advance solution we could use Flask-Migrate extension that handles SQLAlchemy database migrations for Flask applications using Alembic. It can emit ALTER statements to a database in order to change the structure of tables.